packages/wallets/src/components/WalletsCarouselHeader/WalletsCarouselHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useBalance } from '@deriv/api-v2';
+import IcCashierTransfer from '../../public/images/ic-cashier-transfer.svg';
+import { IconButton, WalletText } from '../Base';
+import { WalletCurrencyCard } from '../WalletCurrencyCard';
+import './WalletsCarouselHeader.scss';
+
+type TProps = {
+    balance?: string;
+    currency: string;
+    isDemo?: boolean;
+};
+
+const WalletsCarouselHeader: React.FC<TProps> = ({ balance, currency, isDemo }) => {
+    const history = useHistory();
+    const { isLoading } = useBalance();
+
+    return (
+        <div className='wallets-carousel-header'>
+            <div className='wallets-carousel-header__content'>
+                <WalletCurrencyCard currency={currency} isCarouselHeader isDemo={isDemo} size='md' />
+                <div className='wallets-carousel-header__details'>
+                    <WalletText color='general' size='sm'>
+                        {currency} Wallet
+                    </WalletText>
+                    {isLoading ? (
+                        <div
+                            className='wallets-skeleton wallets-card--balance-loader'
```

Comment:
what is --balance-loader? is it a block, or modifier? 
feels more like a block, so probably wallets-card__balance-loader would be better

Reply:
Updated